corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -35,10 +35,18 @@
   </td>
   <td class="col-sm-2">
     <% if (input == "select1") { %>
-      <select class="query-field form-control"  data-receive="<%- receive %>">
+        <select class="query-field form-control hqwebapp-select2data-receive="<%- receive %>">
```

Comment:
this looks like a bug
```suggestion
        <select class="query-field form-control hqwebapp-select2" data-receive="<%- receive %>">
```